ckanext/oht/plugin.py

```diff
@@ -83,3 +85,41 @@ def get_auth_functions(self):
             'package_collaborator_delete': oht_authz.creators_can_manage_collaborators,
             'package_collaborator_list': oht_authz.creators_can_manage_collaborators
         }
+
+    def identify(self):
```

Comment:
Please add `# IAuthenticator` comment to indacate the section of interface impl.